### PR TITLE
feature/pfdhcplistener-process-readonly-mode

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
-----------
Do not process dhcp packets when the database is in readonly

Impacts
-----------
dhcp processing

NEWS file entries
-------

New Features
++++++++++++
* Do not process dhcp packets when the database is in readonly

Delete branch after merge
-------------------------
NO

### CODE REVIEW

**Comment by louismunro** [comment]
Frankly, this is so small as to not be worth a PR.

File: lib/pf/dhcp/processor.pm [hunk 1]
```diff
@@ -150,6 +150,10 @@ Process a packet
 sub process_packet {
     my $timer = pf::StatsD::Timer->new();
     my ( $self ) = @_;
+    if (db_check_readonly()) {
```
Comment (louismunro) [inline lib/pf/dhcp/processor.pm:153]: Where is that defined?

Comment (jrouzierinverse) [reply lib/pf/dhcp/processor.pm:153]: pf::db

Comment (louismunro) [reply lib/pf/dhcp/processor.pm:153]: ok then.

**Pull request merged**